sky/clouds/service_catalog/constants.py

```diff
@@ -1,6 +1,6 @@
 """Constants used for service catalog."""
 import os
 
-HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/skypilot-org/skypilot-catalog/master/catalogs'  # pylint: disable=line-too-long
+HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/shethhriday29/skypilot-catalog/master/catalogs'  # pylint: disable=line-too-long
```

Comment:
[Quota code](https://github.com/skypilot-org/skypilot-catalog/pull/38) PR has been merged now, we can change this to skypilot-org :)